public/app/boot.js

```diff
@@ -6,5 +6,33 @@
   }).catch(function(err) {
     console.log('Loading app module failed: ', err);
   });
+})();
+
+(function() {
```

Comment:
Please add an overview here to explain why this is required so we don't forget :)